library/std/src/io/mod.rs

```diff
@@ -479,11 +479,11 @@ where
 /// }
 /// ```
 ///
-/// [`read()`]: Read::read
+/// [`read()`]: Self::read
 /// [`&str`]: str
 /// [`std::io`]: self
 /// [`File`]: crate::fs::File
-/// [slice]: ../../std/primitive.slice.html
+/// [slice]: crate::slice
```

Comment:
This links to the module, not the primitive, which doesn't match the text it's used for.

Reply:
I missed it when reviewing on my side, thanks !